corehq/motech/dhis2/entities_helpers.py

```diff
@@ -271,20 +272,28 @@ def create_relationships(
 
     for rel_config in subcase_config.relationships_to_export:
         supercase = get_supercase(subcase_trigger_info, rel_config)
+        if not supercase:
+            # There is no parent case to export a relationship for
+            continue
         supercase_config = get_case_config_for_case_type(
             supercase.type,
             dhis2_entity_config,
         )
+        if not supercase_config:
+            # The parent case type is not configured for integration
+            raise ConfigurationError(_(
```

Comment:
Does this automatically get sent to the users as an email since they will need to know this issue?

Reply:
Yes it does. It gets sent to the addresses listed as "notify addresses" for the ConnectionSettings instance that this repeater uses.